### PR TITLE
blackjack: streamline introduction

### LINKED ISSUE #1556: blackjack: streamline introduction
Firstly, the [introduction.md file](https://github.com/exercism/go/blob/main/exercises/concept/blackjack/.docs/introduction.md#introduction) of the `blackjack` exercise states that a `switch` allows for more "advanced scenarios" than an `if` statement, but it doesn't clearly state _what_ those advanced scenarios are. There are some hints in the comments in the syntax example, but it would likely be easier for the student if the text itself would explain how a `switch` is different from an `if` statement and in what ways.

Secondly, some explicit mention of the `case` and `default` keywords is probably also useful. 

Thirdly, the syntax example uses method invocation syntax (`strings.HasPrefix(value, "val")`), but I don't think students have been taught this concept looking at [the prerequisites](https://github.com/exercism/go/blob/main/config.json#L103). Looking at [the exemplar implementation](https://github.com/exercism/go/blob/main/exercises/concept/blackjack/.meta/exemplar.go#L52), students just need to know that a `switch` can have arbitrary boolean expressions in which for example numbers are compared. I would recommend changing the syntax example to something similar to the exemplar implementation.

### PR DESCRIPTION
**Pull request by andrerfcsantos**
Fixes #1556

Summary of the changes:

 * I revamped the introduction of the concept and the introduction of the exercise as per the suggestions in #1556
 * I also made the `about.md` of the concept to be more comprehensive
 * More friendly link names for the concept

### CODE REVIEW

File: concepts/conditionals-switch/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "blurb": "Switches control the flow of a program based on the value of a variable or expression.",
   "authors": ["tehsphinx"],
-  "contributors": []
+  "contributors": [
+    "andrerfcsantos"
```
Comment (junedev) [inline concepts/conditionals-switch/.meta/config.json:5]: You did so much work for the concept, you should put yourself as author.

File: exercises/concept/blackjack/.docs/introduction.md [hunk 2]
```diff
@@ -1,25 +1,38 @@
...
+case age > 20 && age < 30:
+    // do something if age is between 20 and 30
+case age == 10:
+    // do something if age is equal to 10
+default:
+    // do something else for every other case
 }
 ```
+
+To learn more about this topic, check [Go by Example: Switch][go_by_example_switch] or [Tour of Go: Switch][switch_statement].
```
Comment (junedev) [inline exercises/concept/blackjack/.docs/introduction.md:35]: Please remove this line and the link in line 3 from the exercise introduction. The introduction is "self-sufficient" without them as it should be and we don't really want the student to wander off when they are just about to work on the exercise.

In the concept docs, the links are ok.

**Review by junedev — APPROVED** [review]
Great work! Left 2 minor comments.

**Comment by andrerfcsantos** [comment]
Thanks for the review @junedev.

Applied the suggested changes.

**Pull request merged**